src/Microsoft.Framework.PackageManager/ConsoleCommands/PackagesConsoleCommand.cs

```diff
@@ -0,0 +1,143 @@
+        {
+            packagesCmd.Command("push", c =>
+            {
+                c.Description = "Incremental copy of files from local packages to remote location";
+                var argRemote = c.Argument("[remote]", "Path to remote packages folder");
+                var argSource = c.Argument("[source]",
+                    "Path to source packages folder, default is current directory");
+                c.HelpOption("-?|-h|--help");
+
+                c.OnExecute(() =>
+                {
+                    var reports = reportsFactory.CreateReports(quiet: false);
+
+                    // Implicitly commit changes before push
+                    var commitOptions = new CommitOptions
+                    {
+                        Reports = reports,
+                        SourcePackages = argSource.Value
+                    };
+                    var commitCommand = new CommitCommand(commitOptions);
+                    var success = commitCommand.Execute();
+                    if (!success)
+                    {
+                        return 1;
+                    }
+
+                    var pushOptions = new PushOptions
+                    {
+                        Reports = reports,
+                        SourcePackages = argSource.Value,
+                        RemotePackages = argRemote.Value
+                    };
+                    var pushCommand = new PushCommand(pushOptions);
+                    success = pushCommand.Execute();
+                    return success ? 0 : 1;
+                });
+            });
+        }
+
+        private static void RegisterPullSubcommand(CommandLineApplication packagesCmd, ReportsFactory reportsFactory)
+        {
+            packagesCmd.Command("pull", c =>
+            {
+                c.Description = "Incremental copy of files from remote location to local packages";
+                var argRemote = c.Argument("[remote]", "Path to remote packages folder");
+                var argSource = c.Argument("[source]",
+                    "Path to source packages folder, default is current directory");
+                c.HelpOption("-?|-h|--help");
+
+                c.OnExecute(() =>
+                {
+                    var reports = reportsFactory.CreateReports(quiet: false);
+
+                    bool success;
+                    if (Directory.Exists(argSource.Value))
+                    {
+                        // Implicitly commit changes before pull
+                        var commitOptions = new CommitOptions
+                        {
+                            Reports = reports,
+                            SourcePackages = argSource.Value
+                        };
+                        var commitCommand = new CommitCommand(commitOptions);
+                        success = commitCommand.Execute();
+                        if (!success)
```

Comment:
Use ```return success ? 0 : 1;``` like everywhere else, and move the creation of the variable to right above this line